Go/Go (Embedded Backtick String).sublime-syntax

```diff
@@ -0,0 +1,63 @@
+%YAML 1.2
+---
+name: Go (Embedded Backtick String)
+scope: source.go.embedded-backtick-string
+version: 2
+extends: Packages/Go/Go.sublime-syntax
+hidden: true
+
+contexts:
+  # https://www.jetbrains.com/help/go/using-language-injections.html#use-language-injection-comments
+  inside-comment-magic-language-injection-language-after-language-identifier:
+    - meta_content_scope: comment.line.double-slash.go
+    # TODO: support prefix and suffix
```

Comment:
I've never actually seen these used, and it would likely be not trivial to implement, so I plan to leave this TODO comment here for now.

Reply:
Reasonable.